enhancements/network/multi-network-vrfs.md

```diff
@@ -0,0 +1,924 @@
+independent of networks that other pods may connect to.
+
+## Terminology
+
+* **Primary Network** - The network which is used as the default gateway for the pod. Typically recognized as the eth0
+interface in the pod.
+* **Secondary Network** - An additional network and interface presented to the pod. Typically created as an additional
+Network Attachment Definition (NAD), leveraging Multus.
+* **Cluster Default Network** - This is the routed OVN network that pods attach to by default today as their primary network.
+The pods default route, service access, as well as kubelet probe are all served by the interface (typically eth0) on this network.
+* **User-Defined Network** - A network that may be primary or secondary, but is declared by the user.
+
+## Motivation
+
+As users migrate from OpenStack to Kubernetes, there is a need to provide network parity for those users. In OpenStack,
+each tenant (akin to Kubernetes namespace) by default has a layer 2 network, which is isolated from any other tenant.
+Connectivity to other networks must be specified explicitly as network configuration via a Neutron router. In Kubernetes
+the paradigm is opposite, by default all pods can reach other pods, and security is provided by implementing Network Policy.
+Network Policy can be cumbersome to configure and manage for a large cluster. It also can be limiting as it only matches
+TCP, UDP, and SCTP traffic. Furthermore, large amounts of network policy can cause performance issues in CNIs. With all 
+these factors considered, there is a clear need to address network security in a native fashion, by using networks per
+tenant to isolate traffic.
+
+### User Stories
+
+* As a user I want to be able to migrate applications traditionally on OpenStack to Kubernetes, keeping my tenant network
+  space isolated and having the ability to use a layer 2 network.
+* As a user I want to be able to ensure network security between my namespaces without having to manage and configure
+  complex network policy rules.
+* As an administrator, I want to be able to provision networks to my tenants to ensure their networks and applications
+  are natively isolated from other tenants.
+* As a user, I want to be able to request new private networks for my namespace, without having to get administrator
+  permission.
+* As a user, I want secondary networks to be able to have the same functionality as the primary network, including the
+  ability to use services, and communicate outside of the cluster (north/south).
+* As a user, I want to be able to use my own consistent IP address scheme in my network.
+
+### Goals
+
+* Provide a configurable way to indicate that a pod should be connected to a user-defined network of a specific type as a
+primary interface.
+* The primary network may be configured as a layer 3 or layer 2 type network.
+* Allow networks to have overlapping pod IP address space. This range may not overlap with the default cluster subnet
```

Comment:
it is. I think the word "may" is just confusing for non-english native speakers. It can mean both "possible" or "permission" depending on the context. I'll change the word to be more clear: "This range cannot overlap"